Dockerfile

```
@@ -1,102 +1,103 @@
+#    - Starts from a fresh Elixir slim image.
+#    - Installs only what’s needed at runtime (wget for health checks and CA certs).
+#    - Copies the compiled release and switches to a non-root user.
 ###############################################################################
-FROM elixir:1.18-otp-27-slim AS runtime
+FROM elixir:1.18.3-otp-27-slim AS runtime
 
 WORKDIR /app
 
-# Install wget for health‐checks
+# Install runtime dependencies (wget for health checks, ca-certificates for HTTPS)
 RUN apt-get update \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)